index.html

```diff
@@ -1,383 +1,416 @@
-        <ul>
-          <li>
-            <a href="#">CSS</a>
-          </li>
-          <li>
-            <a href="#">HTML</a>
-          </li>
-          <li>
-            <a href="#">BOOTSTRAP</a>
-          </li>
-          <li>
-            <a href="#">RUBY</a>
-          </li>
-        </ul>
-        <button type="button">See project</button>
-      </article>
-    </section>
+      <section class="headline">
+        <article class="intro">
+          <div class="first_intro_column">
+            <h1 class="main_title">Hey There.<br />I'm Robert.</h1>
+            <h2 class="subtitle">I'm a Software Developer</h2>
+            <p class="desc">
+              I can help you build a product, feature or website. Look through
+              some of my work and experience! if you like what you see and have
+              a project you need coded, don't hesitate to contact me.
+            </p>
+          </div>
+          <div class="second_intro_column">
+            <ul>
+              <li>
+                <a href="https://github.com/Kanyorok">
+                  <span class="icon git"></span>
+                </a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/robert-mbugua-a8745093/">
+                  <span class="icon linked"></span>
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <span class="icon angel"></span>
+                </a>
+              </li>
+              <li>
+                <a href="https://twitter.com/CarlosMbugua">
+                  <span class="icon twitter"></span>
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <span class="icon medium"></span>
+                </a>
+              </li>
+            </ul>
+          </div>
+        </article>
+        <div class="image_show">
+          <img src="icons/Illustration-header.svg" alt="back" />
+        </div>
+      </section>
 
-    <section class="center_content">
-      <article class="content_desc">
-        <h2>Profesional Art <br />Printing Data</h2>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. Has been the industry's standard.
-        </p>
-        <ul>
-          <li>
-            <a href="#">HTML</a>
-          </li>
-          <li>
-            <a href="#">BOOTSTRAP</a>
-          </li>
-          <li>
-            <a href="#">RUBY</a>
-          </li>
-        </ul>
-        <button type="button">See project</button>
-      </article>
-    </section>
+      <section class="works">
+        <header class="works_title">
+          <span> My Recent Works </span>
+          <hr />
+        </header>
+        <article class="works_content">
+          <div class="image_placement">
+            <img src="icons/Works_cont.svg" alt="The Banner" />
+          </div>
+          <div class="text_placement">
```

Comment:
- [ ] This element should be wrapped in a `section` tag and not an `article` tag. The section tag defines a section in a document. A requirement for this project is to use the section tag, not an article tag. kindly remove the article tags to improve your UI.
- [ ] Also try to make sure this section looks exactly like the original design by ensuring you adjust the font size, margins, and padding so that it aligns with the template provided.


> Figma design
<img width="644" alt="Screenshot 2023-05-06 at 13 11 32" src="https://user-images.githubusercontent.com/71140133/236624667-e515dc99-1bb0-4a1d-89ed-3bad166aad2b.png">

> Your design
<img width="1092" alt="Screenshot 2023-05-06 at 13 11 46" src="https://user-images.githubusercontent.com/71140133/236624682-3fcba74a-daf9-4926-90a5-1cd6f86b19c7.png">